Example/Tests/FriendTests/FriendListTests/Repositories/FriendListAPIRepoTest.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  FriendListAPIRepoTest.swift
+//  CommonFiles_Tests
+//
+//  Created by Tiewhan Smith on 2020/04/07.
+//  Copyright © 2020 CocoaPods. All rights reserved.
+//
+
+import XCTest
+
+class FriendListAPIRepoTest: XCTestCase {
+
```

Comment:
Im assuming this class is going to be completed later on, otherwise the empty test class should be removed